### PR TITLE
feat(eslint-config-react-strict): create

### PR DESCRIPTION
**Pull request by ivangabriele**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```
Comment (lionelB) [inline .vscode/settings.json]: its usually not advise to commit editor centric file

Comment (ivangabriele) [reply .vscode/settings.json]: Oops my bad !

File: packages/eslint-config-react-strict/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - "@socialgouv/eslint-config-react-strict"
+```
+
+or add this to your `.eslintrc.json` file:
+
+```json
+{
+  "extends": ["@socialgouv/eslint-config-react-strict"]
+}
+```
```
Comment (lionelB) [inline packages/eslint-config-react-strict/README.md]: usefull to add the pakages.json way too
```json
"eslintConfig": {
  "extends": ["@socialgouv/eslint-config-react-strict"]
}
```

Comment (ivangabriele) [reply packages/eslint-config-react-strict/README.md]: Done ✔️

File: packages/eslint-config-react-strict/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  {
+    "extends": ["@socialgouv/eslint-config-react-strict"]
+  }
+  ```
+
+- or add this to your `package.json` file:
+
+  ```json
+  {
+    "eslintConfig": : {
```
Comment (lionelB) [inline packages/eslint-config-react-strict/README.md]: syntax error  :D

Comment (ivangabriele) [reply packages/eslint-config-react-strict/README.md]: Damn 😄

**Pull request merged**